keras_cv/models/feature_extractor/clip/clip_model_test.py

```diff
@@ -42,98 +42,77 @@ def test_clip_model_golden_values(self):
         processed_image = np.ones(shape=[1, 224, 224, 3])
         processed_text = np.ones(shape=[3, 77])
         attention_mask = np.ones(shape=[3, 77])
-        image_logits, text_logits = model(
+        outputs = model(
             {
-                "image": processed_image,
-                "text": processed_text,
-                "attention_mask": attention_mask,
+                "images": processed_image,
+                "token_ids": processed_text,
+                "padding_mask": attention_mask,
             }
         )
-        self.assertAllClose(image_logits, [[1.896712, 1.896712, 1.896712]])
+
+        # These values are NOT computing using HF as the reference model.
+        # Currently, the numerics of the CLIP model don't match the
+        # HF model exactly (for the same inputs). For the time being,
+        # these tests just confirm that unrelated changed don't affect
+        # the numerics. Once the fix for the numerics is in, we can remove
+        # this comment and the xfail below.
+        self.assertAllClose(
+            outputs["image_logits"], [[10.246354, 10.246353, 10.246354]]
+        )
         self.assertAllClose(
-            text_logits, ops.transpose([[1.896712, 1.896712, 1.896712]])
+            outputs["text_logits"],
+            ops.transpose([[10.246354, 10.246353, 10.246354]]),
         )
 
+        # True reference values computed using HF:
+        # image_logits: [[17.8013, 17.8013, 17.8013]]
+        # text_logits: image_logits.T
+
+        # xfail after assertion
+        pytest.xfail("KerasCV CLIP doesn't match the HF model.")
```

Comment:
@divyashreepathihalli I have included the reference values here computed using the same inputs to the HF model. Currently, `xfail`ing the test to indicate we need to fix this.